src/dmd/backend/cod1.d

```diff
@@ -3111,6 +3111,12 @@ void cdfunc(ref CodeBuilder cdb, elem* e, regm_t* pretregs)
     if (e.EV.E1.Eoper == OPvar)
         sf = e.EV.E1.EV.Vsym;
 
+    /* Assume called function access statics
+     */
+    if (config.exe & (EX_LINUX | EX_LINUX64 | EX_OSX | EX_FREEBSD | EX_FREEBSD64) &&
```

Comment:
There are couple of other OSs that you may consider including: `EX_SOLARIS`, `EX_SOLARIS64`, `EX_OPENBSD`, `EX_OPENBSD64`, `EX_DRAGONFLYBSD64`.
Actually, why not assume TLS access for non-leaf functions for all OSs that support TLS (e.g. including Windows)?

Reply:
Support for them was added by others, in a different way (see backconfig.d), and I have no way to test them.

Reply:
I thought as well, but even so wouldn't it be simpler and safer to always set `accessedTLS` to `true` here, while not adjusting backconfig for other platforms (i.e. enabling the optimization for the platforms on the auto-tester).

Reply:
It's a good thought and I did consider it, but thought it might be confusing when it didn't line up with what backconfig.

As it is, I'm reluctant to say those other platforms are supported when the autotester doesn't test them, and I'm reluctant to change the way code is generated for them.